physical_ai_manager/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
The ".gitignore" file in the top folder of this package, why are you using an extra one?